src/System.Private.CoreLib/shared/System/TimeSpan.cs

```diff
@@ -113,9 +108,9 @@ public double TotalMilliseconds
             }
         }
 
-        public double TotalMinutes => (double)_ticks * MinutesPerTick;
+        public double TotalMinutes => (double)_ticks / TicksPerMinute;
 
-        public double TotalSeconds => (double)_ticks * SecondsPerTick;
+        public double TotalSeconds => (double)_ticks / TicksPerSecond;
```

Comment:
while it improves precision it also makes it slower, 
here is the difference between mulss and divss for my macbook's haswell:
```
vdivss (Latency: 10-20,  R.Throughput: 7-14)
vmulss (Latency:     5,  R.Throughput:  0.5)
```

Reply:
I think ensuring we return an accurate `TotalSeconds` count is generally more important than worrying about a couple of cycles of throughput difference.

If users really have a hot path here and find the loss of precision is acceptable, it is possible for them to do the faster code themselves :) (just get `TimeSpan.Ticks * (1.0 / TimeSpan.TicksPerSecond)`)